### ISSUE TITLE
Option to display tabs on bottom to be more conform with Gnome

### ISSUE THREAD

**Issue by Algram** [horst3180/arc-firefox-theme#31]
Basically like the Adwaita Firefox Skin:

![Image](http://i.imgur.com/NwZtF9A.png)

**Comment by Algram** [comment]
If someone could point me to the file I'd need to edit I would do it myself. Any suggestions?

**Comment by horst3180** [comment]
Hmm, it's probably not an easy change. I'll have a look at it.
The thing is that I would like to keep the tabs on top and it's not possible to change them at runtime.

**Comment by Algram** [comment]
What would be the problem with that? I mean you could just provide another version of the theme, or provide a compiling flag.

**Comment by horst3180** [comment]
Right, I didn't think about a compilation flag. I'll try to implement something like this.

**Comment by Algram** [comment]
That would be great. Maybe take a look at how the Gnome Integration Team did it: https://github.com/gnome-integration-team/firefox-gnome

I already had a look at it, but couldn't find the code that puts tabs on bottom. If you do it would be nice to let me know.

**Comment by horst3180** [comment]
Ah well, I just found this extension https://addons.mozilla.org/de/firefox/addon/tabs-on-bottom/
It works relatively well with the theme. Maybe I can put some specific css for the extension into the theme to make it look a bit better.

**Comment by Algram** [comment]
Hm no not really since it puts the bookmarks completely on top. But there should be the url bar. Also the window controls are on the bottom then which looks weird.

**Comment by ghost** [comment]
Is it possible to provide the code, so we can manually change the theme? Or even better have you made any progress on implementing the bottom tabs option?

**Comment by horst3180** [comment]
Unfortunately this is a pretty invasive change. I can't make this an option at build time.

**Comment by Algram** [comment]
Any updates on this? Can we help?

**Comment by rowanembarks** [comment]
Not even sure why this theme exists without this. It may look like like Arc, but it doesn't integrate Firefox into GNOME at all. The [Adwaita theme](https://github.com/gnome-integration-team/firefox-gnome) for Firefox does a much better job especially when combined with HTitle.
